spring-application/library-client/build.gradle

```diff
@@ -38,7 +38,6 @@ dependencies {
     //implementation project(':library-api')
     implementation('ch.baloise.oidc:library-api:1.0.0-SNAPSHOT')
 
-    implementation('org.springframework.boot:spring-boot-devtools')
```

Comment:
Why?

Reply:
It seemed to be not used - especially not for prod-use. Eventually, it should be added to the developmentOnly dependencies.